### DISCUSSION TITLE
Compact card

### DISCUSSION THREAD

**Discussion by Kayinwonderland** [vasqued2/ha-teamtracker-card#55] [Ideas]
Hey there!

First of all: Thank you so much for this card. I absolutely adore it!
The only Thing that bothers me a tiny bit is the size as it is really huge on my lovelace. Would it be possible to have a more compact card with only basic infos? or probsably a option to tick off things like: Stadium, City of Stadium or to set the font size ( Small, Medium and Large would be okay already)

Greetings

**Comment by vasqued2** [comment]
You should be able to accomplish most, if not everything, you want to do with card-mod.  That's going to be much more powerful than a bunch of one-off configuration options.

I haven't played around with it a lot but the example below shrinks the font size, line height, and changes the font color to red for the whole card.

Then for individual elements it changes the title to green, hides Odds from the top line in the bottom section and then hides the bottom two lines in the PRE card.

Obviously you could do whatever you wanted w/ any element.  These are just examples.

```
card_mod:
  style: |
    ha-card {
     color: red;
     font-size: .7em; 
     line-height: .7em;
    }
    .title {
     color: green;
    }
    .odds {
     display: none;
    }
    .sub2 {
     display: none;
    }
    .sub3 {
     display: none;
    }
```

I'd never looked at the card to optimize the ability to customize it via card-mod so there are probably some things I could do to make it easier like using unique class names across the different game states and paying more attention to the use of fixed pixels for spacing.

If you notice anything that keeps you from doing what you want to do w/ card-mod, let me know and I'll look into changing.

**Comment by vasqued2** [comment]
I’ve made several changes to make this simpler.  It’s in the Refactor branch if you want to take a look.  Changed most items to resize automatically based on font size so makes it easy to scale down.  Also renamed some of the card components to allows hiding items on a PRE card but displaying the same element on the IN card easier.

**Comment by vasqued2** [comment]
I plan to release the changes sometime this weekend.  

Here is the link to some examples on  you to use card-mod to format it.

https://github.com/vasqued2/ha-teamtracker/wiki/Using-CARD-MOD-card-to-format-the-TEAMTRACKER-CARD

**Comment by vasqued2** [comment]
@Kayinwonderland, this is now available in v0.8.0.

Please let me know if it works for you or if you are having any problems doing what you are trying to do.

**Reply by Kayinwonderland** [reply]
Hey! Sorry for my late reply but I was on Holiday. Its fab for me! Much more compact. Thank you so much for directly adopting my feedback ♥

**Reply by vasqued2** [reply]
No worries.  I've been looking for a reason to refactor the card for a while and this was just the nudge I needed.  Thanks for the suggestion.  Glad it met your needs.

**Comment by cableghost** [comment]
I notice in mobile views, the fonts are not small enough... Can we make adjustments to this?

![Screenshot_20231030_091142_Home Assistant.jpg](https://github.com/vasqued2/ha-teamtracker-card/assets/5576447/10f6e9cd-8897-49e7-a854-a66a021ec573)

**Comment by vasqued2** [comment]
Not sure I follow.  You're using card-mod and you want different font sizes depending on whether its mobile or on a computer?
